src/utils/utils.ts

```diff
@@ -17,7 +17,7 @@ export const getModulesForDivision = (
       }
       return {
         ...allModules[k2 as string],
-        slug: `/${division}/${allModules[k2 as string].frontmatter.id}`,
+        slug: `/${division}/${allModules[k2 as string].frontmatter?.id}`,
```

Comment:
```suggestion
        slug: `/${division}/${allModules[k2 as string].frontmatter!.id}`,
```

We probably just want to throw an error here? Otherwise it'll be `/{division}/null` or something (?)